config/environments/development.rb

```diff
@@ -1,4 +1,14 @@
 Rails.application.configure do
+
+    config.action_mailer.smtp_settings = {
+    :address   => "smtp.mandrillapp.com",
+    :port      => 25, # ports 587 and 2525 are also supported with STARTTLS
+    :enable_starttls_auto => true, # detects and uses STARTTLS
```

Comment:
Yes, it did. I used secrets.yml to get the domain, user and password and used de Mandrill tutorial:
https://mandrill.zendesk.com/hc/en-us/articles/205582127-Using-Mandrill-s-SMTP-integration-with-Web-Frameworks

Should I change it? Is there a better option to do it?